src/main/java/item/model/Item.java

```diff
@@ -36,6 +37,17 @@ public double getPrice() {
     return price;
   }
 
+  public int getId() {
+    return id;
+  }
+  public void setIsSponsored(boolean isSponsored) {
```

Comment:
this is not the right way to name a method that returns a boolean.
it should be "setSponsored"

Reply:
for getters, 
it should be "getSponsored" or "isSponsored"